src/tables.css

```diff
@@ -31,7 +31,7 @@
  *
  */
 .table,
-.prose table {
+.prose table:not(.unprose) {
```

Comment:
🤔  ... We'd need to make it clear that you can't nest these things (e.g. prose > unprose > prose > unprose). That's one advantage of applying unprose directly to elements: you can't mess it up. Other reason I lean that way is that I suspect this will *mostly* be used for single elements, right? You have a special link or header, but you still want everything around it to behave as prose. You disagree?